### ISSUE TITLE
Issue: pnpm init empty string argument

### ISSUE THREAD

**Issue by nhu-mai-101** [maoosi/prisma-appsync#162]
pnpm init is running with an empty string as an argument and throwing an error:
```
  shortMessage: 'Command failed with exit code 1: pnpm init ',
  command: 'pnpm init ',
  escapedCommand: 'pnpm init ""',
  exitCode: 1,
```

**Comment by maoosi** [comment]
Thanks for reporting and opening a PR @nhu-mai-101 ! I'll release a patch version soon.

**Comment by maoosi** [comment]
Fix released with [1.0.0-rc.7.2](https://github.com/maoosi/prisma-appsync/releases/tag/1.0.0-rc.7.2).
